### PR TITLE
Fix to use common method for resource bundle Info.plist file generation

### PR DESCRIPTION
**Pull request by dnkoutso**
Same result, no need for a new test since its already covered. Just re-using the same method for consistency.

### CODE REVIEW

**Comment by dnkoutso** [comment]
No changelog intentionally.

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! <br />
You can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:

```markdown
* Fix to use common method for resource bundle Info.plist file  
  [dnkoutso](https://github.com/dnkoutso)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb [hunk 1]
```diff
@@ -343,14 +343,10 @@ def add_resources_bundle_targets
...
                 info_plist_path = path.dirname + "ResourceBundle-#{bundle_name}-#{path.basename}"
-                generator = Generator::InfoPlistFile.new(target.version, target.platform, :bndl)
-                update_changed_file(generator, info_plist_path)
-                add_file_to_support_group(info_plist_path)
+                create_info_plist_file(info_plist_path, bundle_target, target.version, target.platform, :bndl)
 
                 bundle_target.build_configurations.each do |c|
                   c.build_settings['PRODUCT_NAME'] = bundle_name
-                  relative_info_plist_path = info_plist_path.relative_path_from(sandbox.root)
-                  c.build_settings['INFOPLIST_FILE'] = relative_info_plist_path.to_s
```
Comment (dnkoutso) [inline lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb]: This gets set by `create_info_plist_file` invocation above.

**Comment by dnkoutso** [comment]
oops this seems like its updating integration specs with some more output, which is a good thing for verbose.

**Comment by dnkoutso** [comment]
Integration specs https://github.com/CocoaPods/cocoapods-integration-specs/pull/150

**Pull request merged**